packages/arb-ts/token_lists/tokenListTypes.ts

```diff
@@ -0,0 +1,37 @@
+/* from https://github.com/Uniswap/token-lists/blob/master/src/types.ts */
```

Comment:
Should we instead import from `https://www.npmjs.com/package/@uniswap/token-lists`?